### DISCUSSION TITLE
reolink_dev.ptz_control

### DISCUSSION THREAD

**Discussion by SilverApe** [fwestenberg/reolink_dev#428] [Q&A] [answered]
Hi,
I tried to find the answer in the HA forums and tried different ways to make it work but I still don't have the answer.
I want to use the PTZ presets in an automation. The problem is I don't know what to put in the data field.
I used "Call service", as a service I use reolink_dev.ptz_control and in the target field I have put my reolink camera. Now I need to fill in something in "Service data". But what and how? Can anyone help with this?

Thanks in advance!

**Comment by SilverApe** [comment, marked as answer]
I solved it. I had to put TOPOS in service data.

`command: TOPOS
preset: 1`

I hope this helps someone else.
Greetings.

**Reply by mnpg** [reply]
Thanks @SilverApe for the info
